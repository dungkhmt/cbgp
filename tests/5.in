20 20
20 30
12 5
10 0
7 2
17 10
8 5
15 11
15 0
17 9
19 7
16 5
8 2
7 18
4 9
7 1
9 14
18 0
13 0
6 2
7 3
14 17
18 1
11 10
12 4
5 2
2 0
13 4
14 11
12 13
5 13
18 2
1
0 0 21 1 20 3 30 0 14 6 16 4 21 4 27 0 18 5 13 7 9 7 11 9 15 5 12 4 12 8 10 10 21 9 11 7 20 0 23 4 
