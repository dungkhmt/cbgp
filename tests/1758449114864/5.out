(11, 4), (9, -1), (6, 1), (16, 9), (7, 4), (14, 10), (14, -1), (16, 8), (18, 6), (15, 4), (7, 1), (6, 17), (3, 8), (6, 0), (8, 13), (17, -1), (12, -1), (5, 1), (6, 2), (13, 16), (17, 0), (10, 9), (11, 3), (4, 1), (1, -1), (12, 3), (13, 10), (11, 12), (4, 12), (17, 1), 
numberIntersection=0 
minAngle=0.7769949998498014 
sumAngle=16.843728938079522 
minEdgeLength=5.656854249492381 
sumEdgeLength=472.34710166511877 
minDistance=0.014938690445150246 
sumDistance=12700.919456872536 

0.0000000000 0.7769949998 16.8437289381 5.6568542495 472.3471016651 0.0149386904 12700.9194568725
0: (48, 8),
1: (23, 4),
2: (23, 80),
3: (38, 19),
4: (18, 17),
5: (13, 0),
6: (39, 15),
7: (14, 8),
8: (36, 25),
9: (23, 30),
10: (18, 25),
11: (29, 12),
12: (28, 21),
13: (27, 21),
14: (9, 49),
15: (0, 80),
16: (27, 34),
17: (41, 0),
18: (80, 80),
19: (79, 80),
48 8 23 4 23 80 38 19 18 17 13 0 39 15 14 8 36 25 23 30 18 25 29 12 28 21 27 21 9 49 0 80 27 34 41 0 80 80 79 80 
0: (48, 8), 1: (23, 4), 2: (23, 80), 3: (38, 19), 4: (18, 17), 5: (13, 0), 6: (39, 15), 7: (14, 8), 8: (36, 25), 9: (23, 30), 10: (18, 25), 11: (29, 12), 12: (28, 21), 13: (27, 21), 14: (9, 49), 15: (0, 80), 16: (27, 34), 17: (41, 0), 18: (80, 80), 19: (79, 80), 
numberIntersection=0 minAngle=0.7769949998498014 sumAngle=16.84372893806934 minEdgeLength=5.656854249492381 sumEdgeLength=472.3471016651078 minDistance=0.014938690445150246 sumDistance=12700.919456872633 
Time taken: 9.603 s
